CCDFDataModel/CTime.cpp

```diff
@@ -911,26 +911,35 @@ double CTime::quantizeTimeToISO8601(double offsetOrig, CT::string period, CT::st
       int origH = date.hour;
       date.minute = 0;
       date.second = 0;
-      date.hour = origH - origH % H;
+      int restH = origH % H;
+      date.hour = origH - restH;
       offsetLow = thisTime->dateToOffset(date);
-      date.hour = date.hour + H;
+      if (restH > 0) {
+        date.hour = date.hour + restH;
+      }
       offsetHigh = thisTime->dateToOffset(date);
     } else if (hmsPart.indexOf("M") != -1) { // 5M
       hmsPart.replaceSelf("M", "");
       int M = hmsPart.toInt();
       int origM = date.minute;
       date.second = 0;
-      date.minute = origM - origM % M;
+      int restM = origM % M;
+      date.minute = origM - restM;
       offsetLow = thisTime->dateToOffset(date);
-      date.minute = date.minute + M;
+      if (restM != 0) {
```

Comment:
Is there a reason why it's `>0` for the restH comparison, but `!=0` here?

Reply:
Changed this to >0 for consistence